raiden-ts/src/utils/types.ts

```diff
@@ -3,7 +3,8 @@ import * as t from 'io-ts';
 import { BigNumber, bigNumberify, getAddress, isHexString, hexDataLength } from 'ethers/utils';
 import { Two, Zero } from 'ethers/constants';
 import { Either, Right } from 'fp-ts/lib/Either';
-import { ThrowReporter } from 'io-ts/lib/ThrowReporter';
+import { isLeft } from 'fp-ts/lib/These';
```

Comment:
Nitpick: could you import it from `fp-ts/lib/Either` instead, to avoid the added import line, please?